### PR TITLE
UL&S: Re-establishes "signed in" tracking.

### PR DESCRIPTION
**Pull request by diegoreymendez**
WPAuth: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/447

In 15.6 we removed tracking of `signed_in` in some scenarios.  This PR re-establishes tracking `signed_in`.

## To test:

Filter your Xcode console by "Tracked: signed_in".

Test any sign in flow and make sure `signed_in`is tracked when successful.

## PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -395,7 +395,7 @@ PODS:
   - WordPress-Aztec-iOS (1.19.3)
   - WordPress-Editor-iOS (1.19.3):
     - WordPress-Aztec-iOS (= 1.19.3)
-  - WordPressAuthenticator (1.24.1):
+  - WordPressAuthenticator (1.24.0-beta.22):
```
Comment (ScoutHarris) [inline Podfile.lock]: I expect to see your version here (1.24.2). Also, I didn't expect to see other pod changes. Did you branch off of `release/15.7` or `develop`?

Comment (diegoreymendez) [reply Podfile.lock]: Good catch.  I made a mistake when merging the latest from `release/15.7` and apparently picked the lockfile from develop instead.

It's now fixed.

**Review by mindgraffiti — COMMENTED** [review]
See Auth PR

**Review by mindgraffiti — APPROVED** [review]
🚀

**Comment by diegoreymendez** [comment]
I'm closing this as we agreed to only integrating when there are WPiOS side changes.  The WPAuth integration doesn't need to happen now.

**Pull request closed without merging**